src/app/Scenes/Sales/Sales.tsx

```diff
@@ -85,7 +101,24 @@ export const Sales: React.FC = () => {
         testID="Sales-Screen-ScrollView"
         refreshControl={<RefreshControl refreshing={isRefreshing} onRefresh={handleRefresh} />}
       >
-        <Flex py={2} gap={4}>
+        <Flex pb={2} gap={4}>
+          <Box mx={2}>
+            <Text variant="sm-display">
+              Bid on works you love with Artsy’s daily auctions.{" "}
+              <LinkText
+                variant="sm-display"
+                accessibilityRole="link"
+                accessibilityHint="Redirects to Artsy auctions guide"
+                onPress={() => {
+                  trackArticleTap()
+                  navigate(SUPPORT_ARTICLE_URL)
+                }}
+              >
+                Learn more about bidding on Artsy.
+              </LinkText>
+            </Text>
```

Comment:
I intended to extract and follow [some kind of best practice](https://artsy.slack.com/archives/C02BAQ5K7/p1747172054085039?thread_ts=1747170831.300799&cid=C02BAQ5K7) for these hyperlinks, but had to change approach because `RouterLink` seems to introduce a layout glitch when nested inside of a `Text`:

<img height="100" alt="Screenshot 2025-05-14 at 2 35 37 PM" src="https://github.com/user-attachments/assets/6444472b-f633-460b-a3f7-6c73def0bfa4" />

(cc @olerichter00)

Reply:
Yeah, the pattern of wrapping other touchable components with RouterLink isn't ideal. But for now, you could try something like this:

```suggestion
            <>
            <Text variant="sm-display">
              Bid on works you love with Artsy’s daily auctions.{" "}
              </Text>
              <RouterLink to={SUPPORT_ARTICLE_URL} hasChildTouchable onPress={() => {
                  trackArticleTap()
                }}>
              <LinkText
                variant="sm-display"
                accessibilityRole="link"
                accessibilityHint="Redirects to Artsy auctions guide"
              >
                Learn more about bidding on Artsy.
              </LinkText>
            </RouterLink
            </>
```

If this does not work, using `LinkText` without `RouterLink` works as well.

Reply:
Thanks for the suggestion 🙏🏽  

<img width="400" alt="Screenshot 2025-05-15 at 2 55 43 PM" src="https://github.com/user-attachments/assets/c6553116-c953-42c6-9c9a-c54bd1071b15" />

^That creates a new block-level element, rather than the inline element that the design specifies, so I think I'll leave things as-is.